package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "binary-live-api",
-  "version": "18.0.0",
+  "version": "18.0.1",
```

Comment:
x.x.1 is for minor bug fixes
x.1.x is for small new features
1.x.x is for new features

In the end it doesn't matter too much, but still....